js/src/Figure.js

```diff
@@ -80,6 +80,9 @@ var Figure = widgets.DOMWidgetView.extend({
         this.width = impl_dimensions["width"];
         this.height = impl_dimensions["height"];
 
+        this.svg.style("min-width", this.width + "px");
+        this.svg.style("min-height", this.height + "px");
```

Comment:
This seems to be a return to making it so that the svg always grows? What about putting a min-width and min-height for the div in the .less file? Then its overridden by the Layout if desired, but is there if not given in a Layout.